app/locale/en.js

```diff
@@ -4869,6 +4869,7 @@ module.exports = {
       cup: 'Cup',
       blitz: 'Blitz',
       clash: 'Clash',
+      season12_announcement_1: 'Deep in the dungeons, mythical crystals stand tall, collecting energy and mana from abandoned ruins, summon units and cast spells, manage your resources effectively and claim victory!.',
```

Comment:
```suggestion
      season12_announcement_1: 'Deep in the dungeons, mythical crystals stand tall, collecting energy and mana from abandoned ruins. Summon units, cast spells, and claim victory!',
```
Tightening this up a little.